pkg/apis/build/v1beta1/build_types.go

```diff
@@ -200,6 +228,11 @@ type Image struct {
 	//
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
+
+	// VulnerabilityScan references the options for vulnerability scanning
```

Comment:
Pls consider changing this, as is the same as in `VulnerabilityScanOptions`

```suggestion
	// VulnerabilityScan provides configurations about running a scan for your generate image
```